packages/examples/cvat/exchange-oracle/src/core/oracle_events.py

```diff
@@ -44,13 +43,23 @@ class ExchangeOracleEvent_TaskFinished(OracleEvent):
     pass  # escrow is enough for now
 
 
+class ExchangeOracleEvent_EscrowCleaned(OracleEvent):
+    pass  # escrow is enough for now
```

Comment:
```suggestion
    pass
```

With such an event name, I wouldn't expect any extra parameters needed.